docs/guide/object-buildings.md

```diff
@@ -1086,6 +1086,42 @@ SmallResidentialTower {
 
 - `enableBoundingObject`: Defines whether the tower should have a bounding object.
 
+## StripBuilding
+
+A strip of buildings of variable length.
+
+%figure
+
+![StripBuilding](images/objects/buildings/StripBuilding/model.png)
+
+%end
+
+Derived from [Solid](../reference/solid.md).
+
+```
+StripBuilding {
+  SFVec3f    translation           0 0 0
+  SFRotation rotation              0 1 0 0
+  SFString   name                  "strip building"
+  SFFloat    length                100
+  SFInt32    randomSeed            1
+  SFBool     enableBoundingObject  TRUE
+}
+```
+
+> **File location**: "WEBOTS\_HOME/projects/objects/buildings/protos/StripBuilding.proto"
+
+> **License**: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+[More information.](https://cyberbotics.com/webots_assets_license)
+
+### StripBuilding Field Summary
+
+- `length`: Defines the length of the strip.
```

Comment:
We should provide more details here a it is not the exact length, but the minimal length (because we don't cut a building in the middle of it).